tests/pytests/integration/ssh/test_state.py

```diff
@@ -561,3 +562,298 @@ def test_retcode_state_sls_id_render_exception(self, salt_ssh_cli):
     def test_retcode_state_top_run_fail(self, salt_ssh_cli):
         ret = salt_ssh_cli.run("state.top", "top.sls")
         assert ret.returncode == EX_AGGREGATE
+
+
+@pytest.fixture(scope="class")
```

Comment:
```suggestion
@pytest.fixture
```
Class? Here?

Reply:
I couldn't make it module-wide because of the single test module, but this fixture shouldn't need to be reinitialized for each test. Will make it module-wide for the refactoring if that's fine.